sources/osgShadow/ShadowMap.js

```diff
@@ -930,14 +873,6 @@ ShadowMap.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( S
             //console.log( 'shadow early OUT' );
```

Comment:
the comment in aimShadowCastingCamera https://github.com/cedricpinson/osgjs/pull/790/files?diff=split#diff-8cf8f17e8132314edcfa57e87eda7ab4R686 says that it's a frustrum, but the variable is bbox, is it accurate or should be a renamed of variable and comments ?